docs/conf/portland/2025/news/thanks-recap.md

```diff
@@ -56,6 +56,8 @@ We had 1 report of a CoC incident at the conference:
 
 Attendees can always report incidents, even after the conference, by emailing us at [conduct@writethedocs.org](mailto:conduct@writethedocs.org).
 
+![Recap photo](/_static/conf/images/pics/2025/recap-2025.jpg)
+
```

Comment:
Gonna be awkward and say I really don't like the placement here. It does cut off the "Up coming events", and as it already feels like it could be the end of the post I reckon folks won't read that.